clang/lib/Interpreter/DeviceOffload.h

```diff
@@ -39,11 +44,13 @@ class IncrementalCUDADeviceParser : public IncrementalParser {
   ~IncrementalCUDADeviceParser();
 
 protected:
-  IncrementalParser &HostParser;
+  std::unique_ptr<CompilerInstance> DeviceCI;
   int SMVersion;
   llvm::SmallString<1024> PTXCode;
   llvm::SmallVector<char, 1024> FatbinContent;
   llvm::IntrusiveRefCntPtr<llvm::vfs::InMemoryFileSystem> VFS;
+  CodeGenOptions &CodeGenOpts; // intentionally a reference.
```

Comment:
```suggestion
  CodeGenOptions &CodeGenOpts; // Intentionally a reference.
```
Not certain the comment adds much -- we usually don't add reference members accidentally.

Reply:
I think that's important as we are forcibly updating codegen options for the incremental use case.